src/meta/src/hummock/manager/compaction.rs

```diff
@@ -1226,7 +1223,7 @@ impl HummockManager {
 
             {
                 // apply result
-                compact_task.task_status = task.task_status;
+                compact_task.set_task_status(task.task_status);
```

Comment:
Seems unnecessary to have the new `set_task_status` method. Better to just set the field directly.